build.sbt

```diff
@@ -33,11 +30,12 @@ resolvers ++= Seq(
   Resolver.bintrayRepo("webjars", "maven"),
   Resolver.typesafeRepo("releases"),
   Resolver.sbtPluginRepo("releases"),
-  Resolver.sonatypeRepo("releases"),
   Resolver.mavenLocal
 )
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-web" % "1.4.4")
+resolvers ++= Resolver.sonatypeOssRepos("snapshots")
```

Comment:
Why is that snapshots repo needed?

Reply:
Just fix deprecation, I don't know, this repo was here before

Reply:
You can remove it, I am pretty sure